src/ThirdParty/ThirdParty.router.spec.ts

```diff
@@ -93,4 +95,32 @@ describe('ThirdParty router', () => {
         })
     })
   })
+
+  describe('when retreiving the available slots for a third party', () => {
+    let url: string
+    const maxSlots = 10
+    const itemsInCuration = 6
+
+    ;(thirdPartyAPI.fetchMaxItemsByThirdParty as jest.Mock).mockResolvedValueOnce(
+      maxSlots
+    )
+    ;(thirdPartyAPI.isManager as jest.Mock).mockResolvedValueOnce(true)
+    ;(ItemCuration.getItemCurationCountByThirdPartyId as jest.Mock).mockResolvedValueOnce(
+      itemsInCuration
+    )
+    url = '/thirdParties/aThirdPartyId/slots'
```

Comment:
The values should be set in a `beforeEach`, if not, they will not be executed before the `it` clause, but when the file loads.

Reply:
Done!!